src/examples/USEnrichmentExample.cs

```diff
@@ -0,0 +1,87 @@
+namespace Examples
+{
+	using System;
+	using System.Net;
+	using System.Collections.Generic;
+	using System.IO;
+    using System.Linq;
+    using SmartyStreets;
+	using SmartyStreets.USEnrichmentApi;
+    using System.Reflection;
+
+	internal static class USEnrichmentExample
+	{
+		public static void Run()
+		{
+            // specifies the TLS protocoll to use - this is TLS 1.2
+            const SecurityProtocolType tlsProtocol1_2 = (SecurityProtocolType)3072;
+
+            // var authId = "Your SmartyStreets Auth ID here";
+            // var authToken = "Your SmartyStreets Auth Token here";
+
+            // We recommend storing your keys in environment variables instead---it's safer!
+            var authId = Environment.GetEnvironmentVariable("SMARTY_AUTH_ID");
+			var authToken = Environment.GetEnvironmentVariable("SMARTY_AUTH_TOKEN");
+			ServicePointManager.SecurityProtocol = tlsProtocol1_2;
+
+			var client = new ClientBuilder(authId, authToken).BuildUsEnrichmentApiClient();
+			
+			SmartyStreets.USEnrichmentApi.Property.Principal.Result[] results = null;
+            try
+            {
+                results = client.SendPropertyPrincipalLookup("1682393594");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message + ex.StackTrace);
```

Comment:
Add the line "throw" to continue passing the exception upward. If the exception is meant to be ignored then this is fine.